.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
What is the significant of this setting?

Reply:
so that VS code uses the Typescript version in the root `package.json` and not any other version installed in your system